Source/WebCore/platform/PlatformContentFilter.h

```diff
@@ -80,6 +82,10 @@ class PlatformContentFilter : public CanMakeWeakPtr<PlatformContentFilter> {
     void setHostProcessAuditToken(const std::optional<audit_token_t>& token) { m_hostProcessAuditToken = token; }
 #endif
 
+#if HAVE(WEBCONTENTRESTRICTIONS)
+    virtual void setUsesWebContentRestrictions(bool) { };
```

Comment:
Can the Parental Controls content filter check this runtime setting internally instead, since this only applies to the Parental Controls filter?

Reply:
How do we pass down the runtime flag to `ParentalControlsContentFilter`?